bindata/network/openshift-sdn/alert-rules.yaml

```diff
@@ -39,40 +39,39 @@ spec:
       for: 1h
       labels:
         severity: warning
-    - alert: IPTableSyncSDNPod
+    - alert: NodeProxyApplySlow
       annotations:
-        message: SDN pod {{"{{"}} $labels.pod {{"}}"}} on node {{"{{"}} $labels.node {{"}}"}} takes too long to sync iptables rules.
+        message: SDN pod {{"{{"}} $labels.pod {{"}}"}} on node {{"{{"}} $labels.node {{"}}"}} is taking too long to apply iptables rules.
       expr: |
-        histogram_quantile(.95, kubeproxy_sync_proxy_rules_duration_seconds_bucket) * on(pod) group_right kube_pod_info{namespace="openshift-sdn",  pod=~"sdn-[^-]*"} > 15
+        histogram_quantile(.95, kubeproxy_sync_proxy_rules_duration_seconds_bucket) 
+        * on(pod) group_right kube_pod_info{namespace="openshift-sdn",  pod=~"sdn-[^-]*"} > 15
```

Comment:
You can only have one result on the right hand side which isn’t necessarily the case during kube state metrics rollouts. You need to do a topk(1, ...) around your right hand side

Reply:
Ah, that explains some weird alerts. I need that for all kube_pod_*, right?

Reply:
Yes